pkg/cmd/pulumi/install/install.go

```diff
@@ -45,7 +46,9 @@ func NewInstallCmd() *cobra.Command {
 		Short: "Install packages and plugins for the current program or policy pack.",
 		Long: "Install packages and plugins for the current program or policy pack.\n" +
 			"\n" +
-			"This command is used to manually install packages and plugins required by your program or policy pack.",
+			"This command is used to manually install packages and plugins required by your program or policy pack.\n" +
+			"If your Pulumi.yaml file contains a 'packages' section, this command will automatically install\n" +
+			"SDKs for all packages declared in that section.",
```

Comment:
Comment that this is similar to what package address does

Reply:
package address? package add?

Reply:
Add. sorry auto complete